data/cameras.xml

```diff
@@ -15614,6 +15614,27 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
+	<Camera make="FUJIFILM" model="X-A20" supported="unknown-no-samples">
+		<ID make="Fujifilm" model="X-A20">Fujifilm X-A20</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="256" white="4094"/>
+		<Hints>
+			<Hint name="jpeg32_bitorder" value=""/>
+		</Hints>
+		<ColorMatrices>
+			<ColorMatrix planes="3">
+				<ColorMatrixRow plane="0">11540 -4999 -991</ColorMatrixRow>
+				<ColorMatrixRow plane="1">-2949 10963 2278</ColorMatrixRow>
+				<ColorMatrixRow plane="2">-382 1049 5605</ColorMatrixRow>
+			</ColorMatrix>
+		</ColorMatrices>
+	</Camera>
```

Comment:
To be most precise and repeat myself: `supported="unknown-no-samples"` should only include `<ID>`.
```suggestion
	<Camera make="FUJIFILM" model="X-A20" supported="unknown-no-samples">
		<ID make="Fujifilm" model="X-A20">Fujifilm X-A20</ID>
	</Camera>
```

Reply:
As it is really the same camera and same coeffs as X-A10, I'm jus gonna drop this PR then until a sample appears.

Reply:
But that's just it, the sample is more likely to appear with such placeholder entry in place.
I don't understand what the mis-communication here is.

Reply:
I guess I still don't understand what's the utility of all these different labels...

> But that's just it, the sample is more likely to appear

How? I guess we'd have to provide some new messages/user feedback in dt? As things stand currently, there is no difference to plain unknown...

> I don't understand what the mis-communication here is.

I also don't see what's the harm in keeping all the known to be good coeffs for this particular model only, while having it labeled as "unknown-no-samples" to solicit those samples or keep track of what's out there. In my mind would've been just more convenient to flick the switch to support it in the future...

Reply:
> I guess I still don't understand what's the utility of all these different labels...
> 
> > But that's just it, the sample is more likely to appear
> 
> How? I guess we'd have to provide some new messages/user feedback in dt? As things stand currently, there is no difference to plain unknown/no entry...

https://raw.pixls.us/ "[View](https://raw.pixls.us/?missing#repo) all the cameras with no samples at all.",
which parses `cameras.xml`.